v2/stream/new/log.go

```diff
@@ -0,0 +1,35 @@
+package new
+
+import (
+	"log"
+	"os"
+)
+
+type Logger interface {
+	Infof(format string, v ...interface{})
+	Warnf(format string, v ...interface{})
+	Errorf(format string, v ...interface{})
+}
+
+type stdLog struct {
+	logger *log.Logger
+}
+
+var _ Logger = (*stdLog)(nil)
+
+func (s *stdLog) Infof(format string, v ...interface{}) {
+	// NOTE: there is no concept of levels in log
```

Comment:
So basically using the standard logger we only print errors, right?

Reply:
Yes.
This can be changed if you think that's the way to go but I wasn't sure about printing infos/warns by default or using multiple loggers.